modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectRelationshipResourceTest.java

```diff
@@ -15,15 +15,226 @@
+			randomObjectRelationship();
+
+		String externalReferenceCode = RandomTestUtil.randomString();
+
+		randomObjectRelationship.setObjectDefinitionExternalReferenceCode2(
+			externalReferenceCode);
+
+		randomObjectRelationship.setObjectDefinitionId2(() -> null);
+		randomObjectRelationship.setObjectDefinitionModifiable2(() -> null);
+		randomObjectRelationship.setObjectDefinitionSystem2(() -> null);
+
+		ObjectRelationship postObjectRelationship =
+			testPostObjectDefinitionObjectRelationship_addObjectRelationship(
+				randomObjectRelationship);
+
+		Assert.assertNotNull(
+			_objectDefinitionLocalService.
+				getObjectDefinitionByExternalReferenceCode(
+					externalReferenceCode, TestPropsValues.getCompanyId()));
+
+		Assert.assertTrue(
+			postObjectRelationship.getObjectDefinitionModifiable2());
+		Assert.assertFalse(postObjectRelationship.getObjectDefinitionSystem2());
+	}
+
+	@Override
+	protected ObjectRelationship randomObjectRelationship() throws Exception {
+		ObjectRelationship objectRelationship =
+			super.randomObjectRelationship();
+
+		objectRelationship.setName("a" + RandomTestUtil.randomString());
+		objectRelationship.setObjectDefinitionExternalReferenceCode1(
+			_objectDefinition1.getExternalReferenceCode());
+		objectRelationship.setObjectDefinitionExternalReferenceCode2(
+			_objectDefinition2.getExternalReferenceCode());
+		objectRelationship.setObjectDefinitionId1(
+			_objectDefinition1.getObjectDefinitionId());
+		objectRelationship.setObjectDefinitionId2(
+			_objectDefinition2.getObjectDefinitionId());
+		objectRelationship.setObjectDefinitionModifiable2(true);
+		objectRelationship.setObjectDefinitionName2(
+			StringUtil.toLowerCase(RandomTestUtil.randomString()));
+		objectRelationship.setObjectDefinitionSystem2(false);
+		objectRelationship.setParameterObjectFieldId(0L);
+		objectRelationship.setParameterObjectFieldName(() -> null);
+		objectRelationship.setReverse(false);
+		objectRelationship.setType(ObjectRelationship.Type.ONE_TO_MANY);
+
+		return objectRelationship;
+	}
+
+	@Override
+	protected ObjectRelationship
+			testDeleteObjectRelationship_addObjectRelationship()
+		throws Exception {
+
+		return _addObjectRelationship(randomObjectRelationship());
+	}
+
+	@Override
+	protected ObjectRelationship
+			testGetObjectDefinitionByExternalReferenceCodeObjectRelationshipsPage_addObjectRelationship(
+				String externalReferenceCode,
+				ObjectRelationship objectRelationship)
+		throws Exception {
+
+		return _addObjectRelationship(randomObjectRelationship());
+	}
+
+	@Override
+	protected String
+			testGetObjectDefinitionByExternalReferenceCodeObjectRelationshipsPage_getExternalReferenceCode()
+		throws Exception {
+
+		return _objectDefinition1.getExternalReferenceCode();
+	}
+
+	protected Long
+			testGetObjectDefinitionObjectRelationshipsPage_getObjectDefinitionId()
+		throws Exception {
+
+		return _objectDefinition1.getObjectDefinitionId();
+	}
+
+	protected ObjectRelationship
```

Comment:
I believe we need to add the `@Override` annotation to this method, could you please check this?